src/twig/Commands/DefaultCommand.cs

```diff
@@ -52,8 +52,8 @@ public sealed class Settings : CommandSettings
             public bool Verbose { get; set; }
 
             [CommandOption("--advise")]
-            [Description("Find the best compression level given a file and duration")]
-            public int Duration { get; set; }
+            [Description("Find the best compression level given a file and duration\nExample: twig filepath 2000 --advise\nDuration is in milliseconds.")]
```

Comment:
I thought we found a workaround? Or you just did not update the description?

Reply:
oops